src/desktop/apps/artwork/client.tsx

```diff
@@ -53,30 +57,67 @@ const openInquireableModal = (artworkId: string, { ask_specialist }) => {
   })
 }
 
+export const handleOpenAuthModal = options => {
+  mediator.trigger("open:auth", {
+    mode: ModalType.signup,
+    signupIntent: "signup",
+    trigger: "click",
+    destination: location.href,
+    ...options,
+  })
+}
+
 mediator.on("launchInquiryFlow", options => {
-  openInquireableModal(options.artworkId, { ask_specialist: false })
+  if (sd.INQUIRY_AUTH === "experiment" && !sd.CURRENT_USER) {
```

Comment:
#minor might be nice to wrap this line into a re-usable function, its getting used a few times here.